Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.H

```diff
@@ -8,6 +8,10 @@
 #ifndef WARPX_FINITE_DIFFERENCE_SOLVER_H_
 #define WARPX_FINITE_DIFFERENCE_SOLVER_H_
 
+#include <AMReX_MultiFab.H>
+#include "MacroscopicProperties/MacroscopicProperties.H"
+#include "BoundaryConditions/PML.H"
```

Comment:
Just curious: Why are these two additional header files required, with regard to the changes in this file?

Reply:
They're actually not needed :-)